CRM/Discountaddlreg/BAO/PriceFieldValueDiscount.php

```diff
@@ -1,26 +1,28 @@
 <?php
-use CRM_Discountaddlreg_ExtensionUtil as E;
 
+/**
+ * use CRM_Discountaddlreg_ExtensionUtil as E;
```

Comment:
This line should be left unchanged. Civilint is probably saying "Unused `use` statement", right? Technically it's true, this `use` statement is unused in this php file, but it's a standard practice (actually this is done by `civix`) to add this `use` statement in most if not all extension files.  (I've now proposed a patch to fix this in the `civilint` project: https://github.com/civicrm/civicrm-buildkit/issues/535)